src/css/imports/jwplayerelement.less

```diff
@@ -116,6 +116,19 @@
 
 .jw-plugin {
     position: absolute;
+    bottom: 2.5em;
+
+    .jw-banner {
+        max-width: 100%;
+        opacity: 0;
```

Comment:
the opacity is changed to 1 by show function when we want to show the nonlinear ad